web/appsettings.json

```diff
@@ -6,5 +6,11 @@
     }
   },
   "AllowedHosts": "*",
-  "DBConnectionString": "Server=(local);Database=BricksAndHearts;Trusted_Connection=True;Integrated Security=True;"
+  "DBConnectionString": "Server=(local);Database=BricksAndHearts;Trusted_Connection=True;Integrated Security=True;",
+  "Authentication" : {
+    "Google" : {
+      "ClientId" : "",
+      "ClientSecret" : ""
+    }
+  }

```

Comment:
I added empty placeholders so we can see the shape. I haven't set these in Azure yet, or migrated the Azure database